tests/test_nitpicking.py

```diff
@@ -384,7 +384,7 @@ def test_pre_commit_in_subproject(
 
 
                 class ResPartner(models.Model):
-                    _name = "res.partner"
+                    _inherit = "res.partner"
```

Comment:
Just a note @Yajo. I changed this because I needed to test the actual installation of the test module, and I was having some problems with the `_name` assignment.
This shouldn't introduce any other issues, right?

Reply:
Yes, no problem.